examples/example_dotnet/WinSparkleDemoForm.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Reflection;
+using System.Threading;
+using System.Windows.Forms;
+using WinSparkleDotNet;
+
+namespace example_dotnet
+{
+    public partial class WinSparkleDemoForm : Form
+    {
+        private SynchronizationContext _context;
+        private static string _appCastURL = @"http://winsparkle.org/example/appcast.xml";
+
+        public WinSparkleDemoForm()
+        {
+            _context = SynchronizationContext.Current;
+            InitializeComponent();
+            
+            var assembly = Assembly.GetExecutingAssembly();
+            var company = (AssemblyCompanyAttribute)assembly.GetCustomAttributes(typeof(AssemblyCompanyAttribute), false)[0];
+            var name = assembly.GetName().Name;
+            var version = assembly.GetName().Version; //Sparkle can parse basic version scheme like X.X.X.X
+
+            WinSparkle.SetAppcastUrl(_appCastURL);
+            WinSparkle.SetAppDetails(company.Company, name, version.ToString());
```

Comment:
Is this necessary? I.e. is the .NET binding unable to get this info from the assembly automatically? I’d prefer the example to show *simple* use, and this example seems pretty complicated. Similarly with the appcast URL, is there no way to put it into assembly metadata?